index.html

```diff
@@ -55,13 +63,19 @@ <h2 id="supportingtext">I'm a Software Developer</h2>
       </div>
       <div>
         <section class="recentworks">
```

Comment:
Your work section layout is not matching the Figma template on the larger screens. Kindly fix it.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/177906167-0028b512-49d5-40a4-acf6-70630daea110.png) | ![image](https://user-images.githubusercontent.com/33268927/177906044-211a7772-d6db-415e-8359-96fb7da0444c.png) |